### ISSUE TITLE
Site status badge colors

### ISSUE THREAD

**Issue by Clorith** [WordPress/health-check#290]
This is an off-spring from #227, as badges have some considerations ot be made between background and text color (to hit the right contrasts for accessibility, and also to not pick terrible colors).

By providing a good set of default options (perhaps a few more than we need our selves), we also provide the baseline for anything introducing their own tests to ensure they're sticking to accessible colors.

Colors may also indicate severity, so this should be kept in mind when setting up the background/foreground alternatives.

Per the sketches, we've got 4 different types, contrasts need checked on these (as noted by @afercia), and we should also lock down the colors in their entirety.

![site health check status gut 4](https://user-images.githubusercontent.com/6073772/53730139-58999000-3e77-11e9-8b23-5d42634638a6.png)

The above is a sketch, I've been experimenting with some fixed colors, but I've not gotten to checking contrasts yet, but here are the initial values at least:

*blue*
background-color: `#00a0d2`
color: `#ffffff`

*orange*
background-color: `#ffb900`
color: `#000000`

*red*
background-color: `#dc3232`
color: `#ffffff`

*green*
background-color: `#46b450`
color: `#ffffff`

**Comment by hedgefield** [comment]
The blue and green do not pass a contrast test, the other two do.

For blue I'd suggest `#0073AF` (with white text)
For green I'd suggest `#40860A` (with white text)

And a few more suggestions so we have a nice diverse set of colors:

Pink: `#F4B0FC` (with black text)
Light blue: `#10E9FB` (with black text)
Light green: `#60F999` (with black text)
Gray: `#CCCCCC` (with black text)

@afercia does WordPress also find 4.5:1 the minimum, or do they shoot for AAA WCAG?

**Comment by jordesign** [comment]
Those light colour backgrounds are nice alternatives - I really like the look of them.

**Comment by Clorith** [comment]
This was finalized in #309
